### DISCUSSION TITLE
Data groupings

### DISCUSSION THREAD

**Discussion by KarlWallendszus** [cdisc-org/analysis-results-standard#53] [General]
What I understood from the discussion on Friday, and from the current version of the cmap, is that a dataGrouping models both the comparison variable (e.g. treatment arm) and other groupings that might be used in an analysis, e.g. time period, age group, country, etc. I feel the comparison variable is a special case of a dataGrouping as is it used differently in the analysis, so we need a way to distinguish that grouping from any others for a given analysis. I can't see that in the current model.

**Comment by ASL-rmarshall** [comment]
There are currently two types of grouping defined in the model: subject groupings (SubjectGroupingFactor) and dataset record groupings (DataGroupingFactor). Treatment arm would usually be a factor used to group subjects, but (depending on the analysis being performed) there might be situations where treatment arm is used as a factor to group data records.  One of the differences between the two types of grouping would be the associated "n" values - if you wanted to represent % results with reference to the number of subjects in each treatment arm, you'd use a treatment arm subject grouping, but if you wanted to represent your % results with reference to the total number of data records associated with each treatment arm, you'd use a treatment arm data grouping.

Does the difference between subject groupings and data record groupings address the your concern?  If not, could you provide some examples of how treatment arm (as a comparison variable) is used differently in the analysis?  We'll next be working on the modeling for defining operations/methods used within analyses (and the types of result they produce) and it would be interesting to hear if there are situations where groupings affect the analysis in ways other than just subsetting the subjects/data.

**Reply by KarlWallendszus** [reply]
Thanks Richard. I can see that the treatment arm would usually be a subject grouping factor.
It is in the definition of the analysis that the comparison variable will be identified, and as you say, that part of the model is still to be fleshed out. When in is, that should make things clearer, and it is also true that precisely how that is defined may depend on the method of analysis.
